torch/csrc/jit/pickler.h

```diff
@@ -235,7 +235,7 @@ class Unpickler {
         tensor_table_(tensor_table),
         last_opcode_(OpCode::STOP) {}
 
-  std::vector<IValue> parse_ivalue_list();
+  c10::impl::GenericList parse_ivalue_list();
```

Comment:
it's kind of ugly that there are so many places using c10::impl - shall we put `using` declaration somewhere?